hadoop-common-project/hadoop-common/pom.xml

```diff
@@ -382,6 +382,21 @@
       <groupId>com.squareup.okhttp3</groupId>
       <artifactId>mockwebserver</artifactId>
       <scope>test</scope>
+      <exclusions>
```

Comment:
don't these exclusions get picked up from the hadoop project declaration

Reply:
I've made those changes - a new build is running